src/rebar_utils.erl

```diff
@@ -437,6 +437,10 @@ sh_loop(Port, Fun, Acc) ->
             sh_loop(Port, Fun, Fun(Line, Acc));
         {Port, {exit_status, 0}} ->
             {ok, lists:flatten(lists:reverse(Acc))};
+        {Port, {exit_status, 1}} ->
+            {ok, lists:flatten(lists:reverse(Acc))};
+        {Port, {exit_status, 3}} ->
+            {ok, lists:flatten(lists:reverse(Acc))};
```

Comment:
Exit statuses 1 and 3 are very obviously errors in Linux; what you should do for xcopy is to *always* expect an error, but revert it if the code is 1 or 3. Otherwise this patch will break a lot of stuff on unixes, and I assume windows has many commands which do not use 1 and 3 as regular codes, right?